docker.go

```diff
@@ -329,6 +346,33 @@ func OpenMySQLContainerConnection(tries int, delay time.Duration) (c ContainerID
 	return c, nil, errors.New("Could not set up MySQL container.")
 }
 
+// OpenElasticSearchContainerConnection is supported for legacy reasons. Don't use it.
```

Comment:
I think you can remove this method unless you desperately need it. I will remove Open* with v2 release which will be packed through gopkg.